Uplift/Views/GymDetail/Facilities/MiscellaneousInfoView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  MiscellaneousInfoView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/27/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class MiscellaneousInfoView: UIView {
+
+    private let textView = UITextView()
+
+    init(miscellaneousInfo: [String]) {
+        super.init(frame: CGRect.zero)
+
+        let display = formatMiscellaneous(misc: miscellaneousInfo)
+        textView.attributedText = generateAttributedString(for: display, font: UIFont._14MontserratRegular!)
```

Comment:
why do we need to force unwrap here?

Reply:
it gives the "Value of optional type 'UIFont?' must be unwrapped to a value of type 'UIFont'" error without it